### PR TITLE
[Document]Improve quick-start Document

### PR DESCRIPTION
**Pull request by ZuoTiJia**

### CODE REVIEW

**Comment by ZuoTiJia** [comment]
#472

**Comment by zkkxu** [comment]
lgtm

**Review by zkkxu — APPROVED** [review]

**Review by cnoshb — CHANGES_REQUESTED** [review]
Need to add “Use CnosDB” SECTOR in README.md

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Please check [Instructions for joining the group](./docs/guidelines/CnosDBWeChat
 
 ## Contributing
```
Comment (cnoshb) [inline README.md:56]: Add Use CnosDB

**Review by cnoshb — APPROVED** [review]
lgtm

**Pull request merged**